src/lib/components/Footer/FooterBrand.tsx

```diff
@@ -1,28 +1,29 @@
+import classNames from 'classnames';
 import type { ComponentProps, FC, PropsWithChildren } from 'react';
 import { useTheme } from '../Flowbite/ThemeContext';
-
-export interface FooterBrandProps extends Omit<PropsWithChildren<ComponentProps<'div'>>, 'className'> {
+export interface FooterBrandProps extends PropsWithChildren<ComponentProps<'div'>> {
   alt?: string;
   href?: string;
   name?: string;
   src: string;
+  imgClassName?: string;
```

Comment:
This isn't a good idea. You can customize the inner image using CSS, otherwise we will have to add this to a lot of inner components, and it will be a mess.